mitosheet/src/components/taskpanes/Graph/GraphExportTab.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Saga Inc.
+
+import React from 'react';
+import { GraphParams } from '../../../types';
+import MitoAPI from '../../../api';
+import TextButton from '../../elements/TextButton';
+
+/* 
+    The export tab that lets the user copy the graph code or download as a png
+*/
+function GraphExportTab(
+    props: {
+        graphCodeCopied: boolean;
+        _copyGraphCode: () => void;
```

Comment:
I think these are functions that we can push down to the export tab itself. Aka, we don't need these being defined in the highest level of the graphing taskpane anymore.